email-javamail-composer/src/main/java/io/micronaut/email/javamail/composer/DefaultMessageComposer.java

```diff
@@ -54,9 +56,18 @@
  */
 @Singleton
 public class DefaultMessageComposer implements MessageComposer {
-
     public static final String TYPE_TEXT_PLAIN_CHARSET_UTF_8 = "text/plain; charset=UTF-8";
     public static final String TYPE_TEXT_HTML_CHARSET_UTF_8 = "text/html; charset=UTF-8";
+    private static final Logger LOG = LoggerFactory.getLogger(DefaultMessageComposer.class);
+    private static final String SUBTYPE_ALTERNATIVE = "alternative";
+    private static final Map<BodyType, String> BODY_TYPES;
+
+    static {
+        Map<BodyType, String> m = new HashMap<>(2);
+        m.put(BodyType.TEXT, TYPE_TEXT_PLAIN_CHARSET_UTF_8);
+        m.put(BodyType.HTML, TYPE_TEXT_HTML_CHARSET_UTF_8);
+        BODY_TYPES = Collections.unmodifiableMap(m);
+    }
```

Comment:
I can't wait for the next java baseline 😉

Reply:
Maybe, I got to high on my own supply.